webapp/javascript/ui/Table.tsx

```diff
@@ -91,9 +93,14 @@ function Table({
   table,
   tableBodyRef,
   className,
+  isLoading,
```

Comment:
You've added loading state for all tables in the app, @eh-am does it work for web app?
![image](https://user-images.githubusercontent.com/7372044/185909956-c4736878-eb31-4351-8dcb-f2a2b54fd4ae.png)